packages/react-query/src/Elapsed.tsx

```diff
@@ -57,7 +57,7 @@ function getDisplayValue (now = 0, value: BN | Date | number = 0): React.ReactNo
     ? formatValue(elapsed, 's', elapsed < 15)
     : (elapsed < 3600)
       ? formatValue(elapsed / 60, 'min')
```

Comment:
The above we probably want to change to `m` as well.

Reply:
The symbol for the *minute* is **min**, so that's why I left it as such.

Reply:
Just feels inconsistent with "d", "h", "s" and then "min". (I guess I care more about consistency that being "technically correct", SI is _nice_ but in this case we never have "month", so cannot overlap)

Reply:
m could be month...